satpy/readers/fci_l1c_nc.py

```diff
@@ -211,15 +216,36 @@ def __init__(self, filename, filename_info, filetype_info):
 
         self._cache = {}
 
+    @property
+    def nominal_start_time(self):
+        """Get nominal start time."""
+        RC_date = self.filename_info['start_time'].replace(hour=0, minute=0, second=0, microsecond=0)
```

Comment:
looking at this again you could also use `self.observation_start_time` instead of `self.filename_info['start_time']`